deploy/crds/submariner.io_submariners_crd.yaml

```diff
@@ -97,10 +97,14182 @@ spec:
               type: string
             colorCodes:
               type: string
+            engineDaemonSet:
```

Comment:
We really need this 14k LoC?

Reply:
Indeed, we don’t, I was thinking of only including the daemonset status instead.

Reply:
This is generated code - the result of completely embedding a Pod object.  I was a little wary of this too  Perhaps we should embed just the Pod's Status?

Reply:
Sorry my comment crossed with @skitt comment 😄